rs/nns/governance/canister/canister.rs

```diff
@@ -377,6 +377,12 @@ fn list_neurons(req: ListNeurons) -> ListNeuronsResponse {
     governance().list_neurons(&req, caller())
 }
 
+#[query]
+fn get_neuron_index(_req: GetNeuronIndexRequest) -> GetNeuronIndexResponse {
+    debug_log("get_neuron_index");
+    GetNeuronIndexResponse::default()
```

Comment:
A more forceful alternative would be to panic. I don't feel strongly about this. I like it slightly better because it more clearly says "NOT IMPLEMENTED YET", but in practice, nobody is going to start relying on this before it is ready, so it almost certainly doesn't matter...